### ISSUE TITLE
Bug: Can’t change item favicon when its not defined when the item is created without one

### ISSUE THREAD

**Issue by F3000** [strongbox-password-safe/Strongbox#308]
This picture shows the item detail page . Strongbox cant find a favicon for this URL. So the field is empty (red marked).

Is it possible for you that you can display a little placeholder frame in the size of a typical favicon that a user can click on to change the favicon ?

At the moment it is not possible for me to click the red marked area :(

![image](https://user-images.githubusercontent.com/8161283/77123159-a14ec780-6a3f-11ea-8e21-58aca08f921a.jpeg)

**Comment by Slummi** [comment]
I think you are not in editing mode. What do you want to click there?!

**Comment by mmcguill** [comment]
That's super weird @F3000 - I can't reproduce this issue - Can you tell me the steps I need to do to get this problem? This is very important in all issues you open.

**Comment by dan-el** [comment]
@F3000 It looks like you simply need to tap on "Bearbeiten" to switch to edit mode on the entry. Then you should be able to change the icon.

**Comment by georgesnow** [comment]
#Hmm I can’t reproduce either but when I put in random url. I ended up with the DuckDuckGo favicon. It looks like it is grabbing the icon to from the search engine when it fails.

**Comment by mmcguill** [comment]
Going to close this one out, @F3000 - Please file detailed reproduction steps when opening issues.

**Comment by F3000** [comment]
The problem is I can’t reproduce it every time . When I see this effect I wonder why it is so ..... sorry .

**Comment by F3000** [comment]
@mmcguill So now I can describe it better : 

The database was initial created via Strongboxbox . The location is the strongbox  iCloud folder. 

- Open Database in KeepassXC OS X (2.5.3)
- Edit an entry 
- press command+s for saving
- Close KeepassXC 
- Open iOS strongbox and open database
-  When I know look through my list some icons are there , some not . 
- To see all icons again , I have to download all icons and save it again . 


I can only add icons when I re-download all favicons . 
I can’t click in an empty/missing favicon area shown in the picture above

**Comment by dan-el** [comment]
@F3000 can you tap on the empty space after tapping on "Bearbeiten"? Since the "Bearbeiten" button is visible, I assume you are not in editing mode.

**Comment by mmcguill** [comment]
@F3000 - Again, the reproduction steps don't work for me, you will need to start from scratch, e.g. 

1) I created a brand new database in Strongbox/KeePassXC
2) I added a record called x with URL y and added an icon or used Strongbox to grab the FavIcon. It looked ok
3) I then did z.
4) This is a problem because...

Because I just can't action anything here and I can't fix anything here. Having to monitor and respond to these reports consumes really valuable time.

Perhaps @dan-el 's comments will help, I'm not sure because I still don't understand the nature of the issue.
